src/client/commands.ts

```diff
@@ -10,34 +12,43 @@ export function run() {
 /**
  * Build an optimized executable from current file.
  */
-export function prod() {}
+export function prod() { }
```

Comment:
Prettier will handle this. Did you run prettier.

Reply:
I used the wrong formatter.

Reply:
What formatter do you use?

Reply:
In this case the default one. I'm trying to figure out the right settings, so I can add a pull request that automatically sets prettier as the formatter.